mlflow/langchain/langchain_tracer.py

```diff
@@ -1,7 +1,4 @@
-import json
```

Comment:
@serena-ruan Do we have test coverage for the set of information and format produced by MLflowLangChainTracer? Since we're changing the info and the format, I would've expected to see corresponding changes to a test suite. Can we add that coverage?